test/validate_events.test.js

```diff
@@ -51,3 +55,17 @@ describe('Events as given in /_data/events', () => {
     );
   });
 });
+
+describe('Invalid events given in /test/events', () => {
```

Comment:
I think naming that directory `test/invalid_events/` would make it obvious what we are testing here

Reply:
Oh, good catch!  I'll be glad to update with the `invalid_events` directory name.